demos/object_detection_demo/main.cpp

```diff
@@ -201,6 +209,9 @@ int main(int argc, char *argv[]) {
         else if (FLAGS_at == "yolo") {
             model.reset(new ModelYolo3(FLAGS_m, (float)FLAGS_t, FLAGS_auto_resize, FLAGS_yolo_af, (float)FLAGS_iou_t, labels));
         }
+        else if (FLAGS_at == "retina") {
```

Comment:
retinaface better, the output of retinaNet has another format and can not be parsed by retinaface parser if I right remember

Reply:
I named it the same way as @eizamaliev did in [python object_detectin_demo](https://github.com/openvinotoolkit/open_model_zoo/blob/97c02ad141ae6fd76320372a7b1074aa31629f79/demos/python_demos/object_detection_demo/object_detection_demo.py#L47).  I agree, we should be more specific in naming.